clients/admin-ui/src/features/system/SystemInformationFormSelectOptions.ts

```diff
@@ -11,6 +11,10 @@ export const legalBasisForTransferOptions = [
   {
     value: "Adequacy Decision",
     label: "Adequacy decision",
+  },
+    {
+    value: "Supplementary Measures",
+    label: "Supplementary measures",
```

Comment:
i'm just not 100% sure what's gonna happen here because the value coming from the GVL (and from the dictionary) is actually `Supplementary measures` whereas for al the other entries here, the value from the GVL/dctionary matches the `value` field here...

maybe @jpople can clarify what the behavior is gonna be if we get `"Supplementary measures"` populated from the dictionary?

Reply:
I _believe_ it will just fail to populate if it tries to give it an invalid value, but let me test it really quick.  What's the desired behavior here?

Reply:
Should we just change to `Supplementary measures`, `Supplementary measures`?

Reply:
hmm we could probably just update the `value` to be `"Supplementary measures"` just to match what they're giving us. our fides API should allow _anything_ to be submitted now, so that's one option that's probably the simplest to implement.

i think a slightly nicer approach, more in-line with what we've got on the BE now, would be to allow the dictionary to populate _any_ value here and we'd just show exactly what the dictionary gives us if it doesn't match an enumerated "option" here? but then these enumerated options would be all that's allowed for someone _manually_ entering in the values here in the form. that feels like it could be a big change though. it'd make us a bit more resilient to these upstream changes coming in without us knowing, but if it's too much work for now, i think we could just do the first suggestion.

@pattisdr do you agree?

Reply:
whoops, hadn't seen your comment when i wrote mine @pattisdr . i think we landed in the same place, let's just go with that for now 👍

Reply:
updated! let's hope no FE tests fail